Library/Homebrew/formulary.rb

```diff
@@ -314,6 +314,7 @@ def self.load_formula_from_api!(name, flags:)
       end
 
       if (because = json_formula["no_autobump_msg"])
+        because = because.to_sym if NO_AUTOBUMP_REASONS_LIST.key?(because.to_sym)
```

Comment:
How does this relate to the auditor? These seems like a change in non-audit behaviour.